Packs/AWS-Athena/Integrations/AWS-Athena/AWS-Athena.yml

```diff
@@ -2,35 +2,56 @@ commonfields:
   id: AWS - Athena - Beta
   version: -1
 name: AWS - Athena - Beta
```

Comment:
should `Beta` remaining?

Reply:
Yes, we discussed this and decided we want to keep the ID and name (the name must match the ID) like that, and just change the display name, as those are transparent to customers, and changing the ID will require all existing customers to reconfigure a new instance (it will be detected as a new integration).